content/docs/dvclive/get-started.md

```diff
@@ -3,164 +3,189 @@
 DVCLive is a Python library for logging machine learning parameters, metrics and
 other metadata in simple file formats, which is fully compatible with DVC.
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code to start logging
+metrics:
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+<toggle>
+<tab title="Keras">
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback()])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### (Optionally) Log parameters
+</tab>
 
-<toggle>
-<tab title="Single parameter">
+<tab title="Hugging Face">
 
 ```python
-live.log_param("num_classes", 10)
+# train.py
+from dvclive.huggingface import DvcLiveCallback
+
+. . .
+
+ trainer = Trainer(
+    model, args,
+    train_dataset=train_data,
+    eval_dataset=eval_data,
+    tokenizer=tokenizer,
+    compute_metrics=compute_metrics,
+)
+trainer.add_callback(DvcLiveCallback())
+trainer.train()
 ```
 
-See `Live.log_param()`.
-
 </tab>
-<tab title="Multiple parameters">
+<tab title="Pytorch Lightning">
 
 ```python
-params = {
-    "num_classes": 10,
-    "metrics": ["accuracy", "mae"],
-    "optimizer": "adam"
-}
-live.log_params(params)
-```
+# train.py
+from dvclive.lightning import DvcLiveLogger
 
-See `Live.log_params()`.
+. . .
+dvclive_logger = DvcLiveLogger()
+
+trainer = Trainer(logger=dvclive_logger)
+trainer.fit(model)
+```
 
 </tab>
 </toggle>
 
-### Log data
-
-<toggle>
-<tab title="Scalars">
+For other supported frameworks, you can find example snippets and API reference
+on the corresponding page of [ML Frameworks](/doc/dvclive/ml-frameworks).
 
-```python
-live.log("acc", 0.9)
-```
+If you prefer to use the DVCLive API directly, check the
+[API Reference](/doc/dvclive/api-reference).
```

Comment:
Minor: link/mention the basic workflow in the API so it doesn't sound like it's leaving the reader to read through each method.

Reply:
I think we can add a section to the tabs above "No framework" or "Use Directly" ... ? It's worth doing that. I really love these tabs in the get started!

Reply:
I added a tab named `Python API`, not sure if its a good name